pkg/registry/registrytest/etcd.go

```diff
@@ -217,7 +217,8 @@ func (t *Tester) emitObject(obj runtime.Object, action string) error {
 	case etcdstorage.EtcdCreate:
 		err = t.createObject(ctx, obj)
 	case etcdstorage.EtcdDelete:
-		accessor, err := meta.Accessor(obj)
+		var accessor metav1.Object
+		accessor, err = meta.Accessor(obj)
```

Comment:
This would make sense (fix a shadowing bug) if it weren't for the if statement on the line below. This appears to be a no-op as it is.

Reply:
Ah, I see. It is about the subsequent usage (github hid that diff)